nginx.conf

```diff
@@ -0,0 +1,16 @@
+worker_processes auto;
+pid /var/run/nginx/nginx.pid;
+
+events {}
+
+http {
+    server {
+        listen 80;
```

Comment:
Ports Below 1024: On Unix-like systems, ports below 1024 are considered "privileged" ports.
So it's better to use something like 8080